pinax/stripe/tests/test_views.py

```diff
@@ -1,8 +1,11 @@
-from django.core.urlresolvers import reverse
+from django.contrib.auth import get_user_model
```

Comment:
Let's keep this in a `contrib` section of imports after the django imports, please.